diagnostic.md

```diff
@@ -23,7 +27,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$("listerItem").siblings()
```

Comment:
Should be either `$('li').parent().parent().find('form')` or `$('li').parent().parent().children('form');`